moonraker/moonraker.py

```diff
@@ -30,7 +30,7 @@
 
 CORE_PLUGINS = [
     'database', 'file_manager', 'klippy_apis',
-    'machine', 'data_store', 'shell_command']
+    'machine', 'data_store', 'shell_command', 'octoprint_compat']
```

Comment:
I don't think that this should be a core plugin.  My ideal scenario is that slicers will eventually implement support for Moonraker directly, at which point this plugin can be deprecated.  In the meantime users can enable it by adding `[octoprint_compat]` in moonraker.conf.

Reply:
I was considering that.
I decided that since I moved the octoprint compatibility login that was there already, I should keep it as core functionality.

I can change it, but note that users that are currently using the slic3r Octoprint functionality will experience breakage. We might need a transition period to give users enough time to notice that they need to change the config file.

e.g. at least get it in the default config files on both mainsailpi and fluiddpi.

Reply:
I think that is ok.   I can also update config_changes.md after this is merged.